README.md

```diff
@@ -1,25 +1,27 @@
 
-![Lightly Logo](docs/logos/lightly_logo_crop.png)
+![Lightly Logo](docs/logos/lightly_SSL_logo_crop.png)
```

Comment:
Let's make sure that the "alt" tag is Lightly SSL (for SEO).

Maybe by doing this?
```
![Lightly SSL](...)
```

Reply:
Ah this is the github readme, that is not so important.